docs/HowToUseAudio.md

```diff
@@ -27,6 +27,13 @@ Most commonly, audio devices use I2S port for audio data path and
 I2C, SPI etc for audio control path.  
 Depending on the board schematics, a control path (i2c, spi, etc) and a data path (i2s, etc) is to be configured. Please refer [HowToUsePeripheral](HowToUsePeripheral.md)
 
+
+**Driver Naming**
```

Comment:
It doesn't related to only alc5658, you can remove alc5658 here.
Audio Driver registering policy
blah blah blah

Reply:
Okay!

Reply:
Please check yours in markdown view. It looks not good.

Reply:
Okay, I checked the result in a markdown preview app.